.github/scripts/continuous-integration

```diff
@@ -24,8 +24,7 @@ if ($event === '--pull-request')
 {
 	$files = shell_exec('git diff --name-only --diff-filter=ACMRTUXB origin/develop..HEAD');
 }
-else
-{
+else {
```

Comment:
Please revert your changes to the CI script. I think this is a bug with PHP-CS-Fixer.

Reply:
done, and some fails